tests/PHPUnit/Integration/Columns/DimensionTest.php

```diff
@@ -278,14 +278,6 @@ public function test_getId_ShouldCorrectlyGenerateIdFromDimensionsQualifiedClass
         $this->assertEquals("Test.DimensionTest", $this->dimension->getId());
     }
 
-    public function test_factory_ShouldCreateDimensionFromDimensionId()
-    {
-        Manager::getInstance()->loadPlugins(array('ExampleTracker'));
-
-        $dimension = Dimension::factory("ExampleTracker.ExampleDimension");
```

Comment:
Might be useful to transform that into a test of `DimensionsProvider::factory` instead, as that one seems untested.

Reply:
Done 👍